app/frontend/packs/further_conditions/add_another.js

```diff
@@ -0,0 +1,24 @@
+import assignIndexBasedValues from './helper'
+import removeFurtherCondition from './remove'
+
+const addFurtherConditionElement = () => {
+  const items = document.getElementsByClassName('app-add-another__item')
```

Comment:
Is `.app-add-another__item` descriptive enough for the entire document? It seems a bit generic. Perhaps `.app-add-condition*` as a convention?

Also should the base string(s) for these repeatedly defined class names be in an exported constant somewhere?

```js
const CSS_CLASS_ROOT = 'app-add-condition'
const ITEM_CSS_CLASS = CSS_CLASS_ROOT + '__item'
...
```